cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -80,76 +87,106 @@ describe('FluxQueryBuilder', () => {
       cy.getByTestID('tag-selector-key').should('be.visible')
     })
 
-    it.skip('search bar can search fields and tag keys dynamically', () => {
+    it('search bar can search fields and tag keys dynamically', () => {
+      cy.clock(new Date(), ['Date']) // needed by .tick()
+      cy.intercept('POST', '/api/v2/query*').as('query')
+
       // select a bucket
       cy.getByTestID('bucket-selector--dropdown-button').click()
-      cy.getByTestID(`searchable-dropdown--item ${bucketName}`).click()
+      cy.getByTestID(`bucket-selector--dropdown--${bucketName}`).click()
+      cy.wait('@query')
```

Comment:
for all of these waits, we should be watching for the ui to change somehow so that we're decoupling intent from implementation. by leveraging the intercepts, we make the tests more fragile by removing the ability for schema look ups to be done by another rest api, or pulling from a local cache for example. is there any visual dom changes we can be looking at, like a dropdown being populated or a list element showing up?

Reply:
@drdelambre do you mean, instead of waiting on intercepts, we use assertions on visual dom for the changes?